src/tools/profile/read_profile.c

```diff
@@ -152,28 +152,28 @@ static void show_profile_data_accum(profile_data_t *data, options_t *opts)
     qsort(sorted_locations, num_locations, sizeof(*sorted_locations), compare_locations);
 
     /* Print locations */
-    printf("%30s %13s %13s %10s                FILE     FUNCTION\n",
+    printf("%35s %13s %13s %10s                FILE     FUNCTION\n",
```

Comment:
what do you think to define macro for this value:
```
#define UCS_PROFILE_MAX_NAME_LEN   35
```

and update `UCS_PROFILE_*` macros to check that the name is `<=` than `UCS_PROFILE_NAME_FORMAT`, e.g.
```
#define UCS_PROFILE_FUNC(_ret_type, _name, _arglist, ...) \
    static UCS_F_ALWAYS_INLINE _ret_type _name##_inner(__VA_ARGS__); \
    \
    _ret_type _name(__VA_ARGS__) { \
        ucs_assert(strlen(_name) <= UCS_PROFILE_MAX_NAME_LEN); \
        UCS_PROFILE_SCOPE_BEGIN(); \
        _ret_type _ret = _name##_inner _arglist; \
        UCS_PROFILE_SCOPE_END(#_name); \
        return _ret; \
    } \
    static UCS_F_ALWAYS_INLINE _ret_type _name##_inner(__VA_ARGS__)

```

So, if the newly added function name is greater than this value, a developer could change `UCS_PROFILE_MAX_NAME_LEN` to the new value to fit new name

Reply:
sounds, good. But maybe we can just cut the name by MAX_NAME_LEN to keep the table format accurate?
I guess 35 symbols would be enough to recognize the func.

Reply:
> But maybe we can just cut the name by MAX_NAME_LEN to keep the table format accurate?

yes, good idea :)